core/types/header_ext_test.go

```diff
@@ -117,22 +118,35 @@ func headerWithNonZeroFields() (*Header, *HeaderExtra) {
 	return header, extra
 }
 
-func allExportedFieldsSet[T interface {
+func allFieldsSet[T interface {
 	Header | HeaderExtra | Block | Body | BlockBodyExtra
 }](t *testing.T, x *T, ignoredFields ...string) {
+	t.Helper()
```

Comment:
I think `t.Helper()` here isn't very useful because it will report all errors as being at 78/79 instead of where they're actually occurring.

Reply:
Removed since it was actually ineffective given we run subtests in this function (TIL/TIN this!)